src/commands/org/create/sandbox.ts

```diff
@@ -176,7 +200,14 @@ export default class CreateSandbox extends SandboxCommandBase<SandboxCommandResp
       tracksSource: this.flags['no-track-source'] === true ? false : undefined,
     });
     const sandboxReq = await this.createSandboxRequest();
-    await this.confirmSandboxReq({ ...sandboxReq, ...(this.flags.clone ? { CloneSource: this.flags.clone } : {}) });
+    await this.confirmSandboxReq({
+      ...sandboxReq,
+      ...(this.flags['source-sandbox-name']
+        ? { CloneSource: this.flags['source-sandbox-name'] }
+        : this.flags['source-id']
+        ? { CloneSource: this.flags['source-id'] }
+        : {}),
```

Comment:
`CloneSource` will only be shown if when using the source flags, what if I'm specifying the ID in the def. file instead?

Maybe `CloneSource` should always show the source ID (you can grab it from `sandboxReq.SourceId`

Reply:
@soridalac this is still unresolved ⬆️ 
we can ignore `--source-sandbox-name`, just check for `sandboxReq.SourceId`